main.go

```diff
@@ -102,6 +102,10 @@ func main() {
 		logFail("Issue with input: %s", err)
 	}
 
+	if len(configs.NewVersionCode) < 1 && len(configs.NewVersionName) < 1 {
```

Comment:
Please do this check in `configs.validate()`.

Please never use string len check this way. Either it should be `len(configs.NewVersionCode) == 0` but better to check simply `configs.NewVersionCode != ""`. For the latter we already have the input package's ValidateIfNotEmpty or something similar.